USAGE.md

```diff
@@ -74,16 +61,44 @@ The GDL performs many checks on datasets. If any of the checks fail the dataset
 - Every metadata and asset URL must be readable by the GDL.
 - A dataset *may* refer to the same asset more than once. All references to the same asset must have the same multihash. That is, having a SHA-1 and a SHA-256 checksum for the same file will be considered invalid, even if both checksums are valid. This is to enable a simpler checksum validation.
 
-## Data Lake Endpoints Usage
+
+# Authentication and authorization
+Data Lake allows read/write access for users authorized by SAML identity provider
+(`DATALAKE_SAML_IDENTITY_PROVIDER_ARN`) configured during deployment time (see
+[README](README.md#aws-infrastructure-deployment-cdk-stack)).
+
+Example of AWS service account authentication and authorization in to Data Lake users role via
+Azure:
+
+- `~/.aws/config` file
+    ```
+    [profile data-lake-users]
+    azure_tenant_id=<AZURE_TENANT_ID>
+    azure_app_id_uri=https://signin.aws.amazon.com/saml\#<AZURE_APP_ID>
+    azure_default_username=<AZURE_USER_NAME>
+    azure_default_role_arn=arn:aws:iam::<DATALAKE_AWS_ACCOUNT_ID>:role/<DATALAKE_USER_ROLE_NAME>
```

Comment:
This script  auto populates the aws config from https://myapplications.microsoft.com/
https://confluence.linz.govt.nz/display/GEOD/Login+to+AWS+Service+Accounts+via+Azure+in+Command+Line

So could maybe change the usage guidance could mirror that to avoid confusion?

Something like:

```
[profile li-datalake-prod]
azure_tenant_id=<AZURE_TENANT_ID>
azure_app_id_uri=<AZURE_APP_ID>
azure_default_username=<AZURE_USER_NAME>
azure_default_role_arn=
azure_default_remember_me=true
azure_default_duration_hours=12
```

And then 

```bash
aws-azure-login --profile li-datalake-prod
```

Reply:
It means they'll have to select which role they want in the CLI (if they have more than one), but probably not a big issue.

Reply:
@billgeo , even after reading your comments multiple times I don't understand what do you mean. Do you want me just keep empty value for `azure_default_role_arn` ?

Reply:
Yes, I just have an empty value for `azure_default_role_arn` and it works fine.

I followed the instructions in that confluence page and it auto populates the aws config file with the value empty. In practice, this is what most LINZ users will be doing I think, so I thought it would be good to have this usage example the same as what that confluence page script does.

Reply:
Instructions updated to what we have in Confluence, thanks